packages/office-ui-fabric-react/src/components/Image/examples/Image.None.Example.tsx

```diff
@@ -19,11 +19,11 @@ export class ImageNoneExample extends React.Component<any, any> {
       <div>
         <p>By setting the imageFit property to "none", the image will remain at its natural size, even if the frame is made larger or smaller by setting the width and height props.</p>
         <Label>The image is larger than the frame, so it is cropped to fit. The image is positioned at the upper left of the frame.</Label>
-        <Image  { ...imageProps as any } />
+        <Image  { ...imageProps as any } alt='larger image fit none example' />
         <br />
```

Comment:
Let's be a bit more verbose with these descriptions as i'm not entirely sure that non-sighted user would understand these. Maybe something like 

```
Example implementation of the property imageFit using the none value.
```